sumologic/sumologic_client.go

```diff
@@ -60,6 +61,13 @@ func createNewRequest(method, url string, body io.Reader, accessID string, acces
 	return req, nil
 }
 
+func logRequestAndResponse(req *http.Request, resp *http.Response) {
+	var maskedHeader = req.Header.Clone()
+	maskedHeader.Set("Authorization", "xxxxxxxxxxx")
+	log.Printf("[DEBUG] Request made to Sumo Logic: [Method=%s] [URL=%s] [Headers=%s]", req.Method, req.URL, maskedHeader)
+	log.Printf("[DEBUG] Response received from Sumo Logic: [StatusCode=%s]", resp.Status)
```

Comment:
Similarly, here we can say "Received response: ". 

Since, we are just logging the response, we should just log it with the request. Otherwise, in a multithreaded setup, it would hard to correlate a request to the corresponding response.